### ISSUE TITLE
Specifying MagmaSat model creates type issues

### ISSUE THREAD

**Issue by fboschetty** [kaylai/VESIcal#179]
I'm using VESIcal to calculate saturation pressures and equilibrium fluid compositions locally, i've installed thermoengine.
When I specify any model but magmasat the commands run no problem.
As soon as I specify model="MagmaSat", the returned pressure from calculate_saturation_pressure() is an incompatible data type for the subsequent calculate_equilibrium_fluid_comp():

```
SatP_bar = v.calculate_saturation_pressure(sample=mysample, temperature=TC, model=Model).result
fluid = v.calculate_equilibrium_fluid_comp(sample=mysample, temperature=TC, pressure=SatP_bar, model=Model).result
```
Gives:
VESIcal.core.InputError: presure must be type float or int

**Comment by FelipeKreulich** [comment]
Have you seen what this model returns? From what I saw, it returns a boolean but it expects a float or int, maybe it should convert to int or float

**Comment by FelipeKreulich** [comment]
Have you tried to visualize what kind of data it returns? From what I saw, it returns a boolean but it expects a float or int, maybe it should convert to int or float

**Comment by fboschetty** [comment]
Yes, it's a np.int64. I can just convert it to float and it works fine but I just wanted to point out that the typing changes depending on what model I specify. To ensure they are all interchangeable this probably ought to be looked in to.

**Comment by FelipeKreulich** [comment]
oh yes, that's right, it's something to really analyze.

**Comment by kaylai** [comment]
Hi, thank you for pointing this out. MagmaSat is coded differently than all of the other models since it relies on ENKI thermoengine, so I'm sure that's the source of the inconsistency. This should be an easy fix, which I promise to get to very soon, when I have more time with my computer!

**Comment by kaylai** [comment]
This is finally fixed in the latest version 1.2.5! 30e65e2bd8013c5f49f23af6eb738e4af6899907
